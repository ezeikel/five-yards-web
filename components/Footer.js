import Link from "next/link";
import styled from "styled-components";
import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
import Logo from "./svgs/Logo";
import GDPRLogo from "./GDPRLogo";

const Wrapper = styled.footer`
  display: flex;
  flex-direction: column;
  background-color: var(--color-black);
  color: var(--color-white);
  padding: var(--spacing-large);
  font-size: 14px;
  @media (min-width: 768px) {
    padding: var(--spacing-huge);
    > div {
      &:first-of-type {
        display: flex;
      }
      > div {
        & + div {
          margin-left: var(--spacing-huge);
        }
      }
    }
  }
`;

const LogoWrapper = styled.div`
  display: flex;
  justify-content: center;
  margin-bottom: var(--spacing-medium);
  svg {
    width: 86px;
    height: 73px;
  }
  @media (min-width: 768px) {
    margin-bottom: 0;
    svg {
      width: 132px;
      height: 113px;
    }
  }
  @media (min-width: 1280px) {
    margin-bottom: 0;
    svg {
      width: 138px;
      height: 118px;
    }
  }
`;

const Links = styled.div`
  display: flex;
  margin-bottom: var(--spacing-large);
  > div {
    flex: 0 1 50%;
    + div {
      margin-left: var(--spacing-large);
    }
    > ul {
      li + li {
        margin-top: var(--spacing-small);
      }
    }
    > span {
      display: flex;
      font-size: 20px;
      font-weight: 700;
      margin-bottom: var(--spacing-medium);
    }
    a {
      color: var(--color-white);
    }
  }
  @media (min-width: 768px) {
    margin-bottom: 0;
    > div {
      flex: 0 1 auto;
      + div {
        margin-left: var(--spacing-huge);
      }
      > span {
        font-size: 20px;
      }
      > ul li {
        font-size: 14px;
      }
    }
  }
  @media (min-width: 1280px) {
    > div {
      > ul li {
        font-size: 16px;
      }
    }
  }
`;

const Social = styled.div`
  display: flex;
  margin-bottom: var(--spacing-large);
  > div {
    flex: 0 1 50%;
    + div {
      margin-left: var(--spacing-large);
    }
    &:first-of-type {
      > span {
        display: flex;
        font-size: 20px;
        font-weight: 700;
        margin-bottom: var(--spacing-medium);
      }
      > ul {
        display: flex;
        li + li {
          margin-left: var(--spacing-medium);
        }
      }
    }
    &:last-of-type {
      svg {
        width: 105px;
        height: 39px;
      }
    }
  }
  @media (min-width: 768px) {
    margin-bottom: 0;
    > div {
      flex: 0 1 auto;
      + div {
        margin-left: var(--spacing-huge);
      }
      &:first-of-type {
        > span {
          font-size: 20px;
        }
      }
      &:last-of-type {
        svg {
          width: 142px;
          height: 53px;
        }
      }
    }
  }
  @media (min-width: 1280px) {
    > div {
      &:last-of-type {
        svg {
          width: 213px;
          height: 80px;
        }
      }
    }
  }
`;

const Copyright = styled.div`
  color: #bebebe;
  align-self: flex-end;
`;

const Footer = () => (
  <Wrapper>
    <div>
      <LogoWrapper>
        <Logo />
      </LogoWrapper>
      <Links>
        <div>
          <span>Join us</span>
          <ul>
            <li>
              <Link href="/">Tailor sign up</Link>
            </li>
            <li>
              <Link href="/">Fabric seller sign up</Link>
            </li>
          </ul>
        </div>
        <div>
          <span>Legal</span>
          <ul>
            <li>
              <Link href="/">Terms and conditions</Link>
            </li>
            <li>
              <Link href="/">Privacy</Link>
            </li>
            <li>
              <Link href="/">Cookies</Link>
            </li>
            <li>
              <Link href="/">Site map</Link>
            </li>
          </ul>
        </div>
      </Links>
      <Social>
        <div>
          <span>Follow us</span>
          <ul>
            <li>
              <a href="https://www.facebook.com/fiveyardsapp">
                <FontAwesomeIcon
                  icon={["fab", "facebook-f"]}
                  color="var(--color-white)"
                  size="2x"
                />
              </a>
            </li>
            <li>
              <a href="https://twitter.com/fiveyardsapp">
                <FontAwesomeIcon
                  icon={["fab", "twitter"]}
                  color="var(--color-white)"
                  size="2x"
                />
              </a>
            </li>
            <li>
              <a href="https://www.instagram.com/fiveyardsapp">
                <FontAwesomeIcon
                  icon={["fab", "instagram"]}
                  color="var(--color-white)"
                  size="2x"
                />
              </a>
            </li>
          </ul>
        </div>
        <div>
          <GDPRLogo />
        </div>
      </Social>
    </div>
    <Copyright>&copy; {new Date().getFullYear()} Five Yards</Copyright>
  </Wrapper>
);

export default Footer;
