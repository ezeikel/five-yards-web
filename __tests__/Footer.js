import { render } from "@testing-library/react";
import Footer from "../components/Footer";

test("should render", () => {
  const { container } = render(<Footer />);

  expect(container).toMatchSnapshot();
});
