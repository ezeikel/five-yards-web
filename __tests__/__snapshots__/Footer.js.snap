// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  background-color: var(--color-black);
  color: var(--color-white);
  padding: var(--spacing-large);
  font-size: 14px;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  margin-bottom: var(--spacing-medium);
}

.c1 svg {
  width: 86px;
  height: 73px;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-bottom: var(--spacing-large);
}

.c2 > div {
  -webkit-flex: 0 1 50%;
  -ms-flex: 0 1 50%;
  flex: 0 1 50%;
}

.c2 > div + div {
  margin-left: var(--spacing-large);
}

.c2 > div > ul li + li {
  margin-top: var(--spacing-small);
}

.c2 > div > span {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-size: 20px;
  font-weight: 700;
  margin-bottom: var(--spacing-medium);
}

.c2 > div a {
  color: var(--color-white);
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-bottom: var(--spacing-large);
}

.c3 > div {
  -webkit-flex: 0 1 50%;
  -ms-flex: 0 1 50%;
  flex: 0 1 50%;
}

.c3 > div + div {
  margin-left: var(--spacing-large);
}

.c3 > div:first-of-type > span {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-size: 20px;
  font-weight: 700;
  margin-bottom: var(--spacing-medium);
}

.c3 > div:first-of-type > ul {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c3 > div:first-of-type > ul li + li {
  margin-left: var(--spacing-medium);
}

.c3 > div:last-of-type svg {
  width: 105px;
  height: 39px;
}

.c4 {
  color: #bebebe;
  -webkit-align-self: flex-end;
  -ms-flex-item-align: end;
  align-self: flex-end;
}

@media (min-width:768px) {
  .c0 {
    padding: var(--spacing-huge);
  }

  .c0 > div:first-of-type {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
  }

  .c0 > div > div + div {
    margin-left: var(--spacing-huge);
  }
}

@media (min-width:768px) {
  .c1 {
    margin-bottom: 0;
  }

  .c1 svg {
    width: 132px;
    height: 113px;
  }
}

@media (min-width:1280px) {
  .c1 {
    margin-bottom: 0;
  }

  .c1 svg {
    width: 138px;
    height: 118px;
  }
}

@media (min-width:768px) {
  .c2 {
    margin-bottom: 0;
  }

  .c2 > div {
    -webkit-flex: 0 1 auto;
    -ms-flex: 0 1 auto;
    flex: 0 1 auto;
  }

  .c2 > div + div {
    margin-left: var(--spacing-huge);
  }

  .c2 > div > span {
    font-size: 20px;
  }

  .c2 > div > ul li {
    font-size: 14px;
  }
}

@media (min-width:1280px) {
  .c2 > div > ul li {
    font-size: 16px;
  }
}

@media (min-width:768px) {
  .c3 {
    margin-bottom: 0;
  }

  .c3 > div {
    -webkit-flex: 0 1 auto;
    -ms-flex: 0 1 auto;
    flex: 0 1 auto;
  }

  .c3 > div + div {
    margin-left: var(--spacing-huge);
  }

  .c3 > div:first-of-type > span {
    font-size: 20px;
  }

  .c3 > div:last-of-type svg {
    width: 142px;
    height: 53px;
  }
}

@media (min-width:1280px) {
  .c3 > div:last-of-type svg {
    width: 213px;
    height: 80px;
  }
}

<div>
  <footer
    class="c0"
  >
    <div>
      <div
        class="c1"
      >
        <svg
          viewBox="0 0 57.56 50.82"
          xmlns="http://www.w3.org/2000/svg"
        >
          <g>
            <g
              fill="#ffffff"
            >
              <path
                d="M0 21l2.17-4.61 1.83.86-1.16 2.48 2.15 1 1-2.17 1.84.86-1 2.17 3.68 1.73-1 2.13zM5.4 15.37a1.19 1.19 0 01-1-.24 1.23 1.23 0 01-.52-.85 1.37 1.37 0 01.28-1 1.35 1.35 0 01.88-.56 1.25 1.25 0 011 .24 1.21 1.21 0 01.5.86 1.3 1.3 0 01-.28 1 1.33 1.33 0 01-.86.55zm6.6 5.19l-5.59-4.17 1.42-1.92 5.62 4.17zM17.37 14.87l-2.74 2.39-5.81-4.21 1.74-1.52 4.59 3.56-2.92-5L14 8.55zM23.76 8.32l-4.54 2a1.31 1.31 0 001.79.56c.54-.24.8-.58.79-1L23.89 9a3.19 3.19 0 01-.38 2.16 3.75 3.75 0 01-1.79 1.47 3.49 3.49 0 01-2.79.12 3.57 3.57 0 01-2-2 3.56 3.56 0 01-.13-2.8 3.52 3.52 0 012-2 3.49 3.49 0 012.79-.12 3.6 3.6 0 012 2c.02.08.08.26.17.49zm-5 .91l2.49-1.1a1.27 1.27 0 00-1.71-.52 1.29 1.29 0 00-.79 1.62zM31.71 10.53h-2.39l-.05-3.43L27 .11h2.27l1.16 4.26 1-4.29h2.27L31.66 7.1zM37.45 10.41a2.27 2.27 0 01-2.48.65 2.69 2.69 0 01-1-.46 1.89 1.89 0 01-.63-.85 1.86 1.86 0 010-1.15 2 2 0 01.41-.83 1.42 1.42 0 01.62-.46 3.39 3.39 0 01.76-.17 3.75 3.75 0 01.82 0c.27 0 .54.06.8.07h.71a1.1 1.1 0 00.54-.06.48.48 0 00.28-.34.65.65 0 00-.12-.57 1.11 1.11 0 00-.6-.35.87.87 0 00-1.18.67L34.17 6a2.5 2.5 0 01.42-.9 2.16 2.16 0 01.66-.59 2.84 2.84 0 01.85-.29 3.35 3.35 0 01.95-.05 5.7 5.7 0 011.9.48 3.06 3.06 0 01.78.47 2.33 2.33 0 01.57.61 2 2 0 01.29.78 2.24 2.24 0 01-.06.92L39.37 12l-2.19-.56zm-1.73-1a.9.9 0 00.5.28 1.28 1.28 0 001-.08 1.19 1.19 0 00.56-.8l.1-.41a2.58 2.58 0 01-.68.07h-.7a1.57 1.57 0 00-.54.09.51.51 0 00-.31.36.52.52 0 00.07.53zM42 13.61l-2.07-1.19 3.47-6.07 1.95 1.11-.64 1.12A2.45 2.45 0 0146 8.09a2.23 2.23 0 011.29.31 1.23 1.23 0 01.3.2l-1.15 2a2.25 2.25 0 00-.58-.47 1.76 1.76 0 00-1.22-.27 1.27 1.27 0 00-.88.69zM48.25 17.61a2.53 2.53 0 01-2.2-.78 2.89 2.89 0 01-.89-2.33 3.87 3.87 0 011.29-2.5A3.91 3.91 0 0149 10.8a2.91 2.91 0 012.31.95 2.54 2.54 0 01.69 2.12l3.09-3 1.66 1.71-7.52 7.33-1.62-1.65zm.9-4.43a1.81 1.81 0 00-1.62 1.58 1.36 1.36 0 00.41 1 1.34 1.34 0 001 .44 1.79 1.79 0 001.62-1.58 1.39 1.39 0 00-.41-1.05 1.36 1.36 0 00-1-.39zM52.2 24.93a3.88 3.88 0 01-1.52-1.72 4 4 0 01-.45-2.41 2.07 2.07 0 011.2-1.62l1 1.95a.8.8 0 00-.48.57 1.27 1.27 0 00.13.81 1.13 1.13 0 00.42.5.5.5 0 00.5 0 .43.43 0 00.26-.37 1.53 1.53 0 000-.62c-.06-.23-.13-.48-.21-.76a5 5 0 01-.2-.85 3.51 3.51 0 010-.85 1.58 1.58 0 01.32-.79 2 2 0 01.81-.65 1.83 1.83 0 011.8 0 3.56 3.56 0 011.47 1.64 3.61 3.61 0 01.38 2.32 2.07 2.07 0 01-1.12 1.56l-1-2a.77.77 0 00.42-.51.92.92 0 00-.08-.7 1 1 0 00-.39-.47.45.45 0 00-.46 0 .41.41 0 00-.22.3 1.2 1.2 0 000 .49c.05.19.1.41.17.65s.13.48.2.72a2.82 2.82 0 01.13.77 3.08 3.08 0 01-.06.75 1.53 1.53 0 01-.35.68 2.16 2.16 0 01-.74.56 1.94 1.94 0 01-1.93.05zM47.65 25.63c-.93-2.24-1.86-4.73-3.55-6.42a19.58 19.58 0 00-6.3-3.84 19.13 19.13 0 00-7.18-1.51c-2.49 0-4.81 1.07-7 2a18.28 18.28 0 00-6 3.83 18.65 18.65 0 00-4.24 5.87 18.8 18.8 0 00-1 7.16 18.54 18.54 0 001.13 7.11 19.07 19.07 0 003.94 6.13c1.7 1.69 4.11 2.45 6.36 3.38a17.46 17.46 0 006.88 1.31c2.49 0 5.15.61 7.31-.29s4.09-2.86 5.78-4.56 2.46-4 3.4-6.26S49 35.18 49 32.69a18.41 18.41 0 00-1.35-7.06zm-2.36 13.13a16.31 16.31 0 01-3.18 5.41A16.92 16.92 0 0137 48c-1.9.79-4.15.26-6.34.26a15.68 15.68 0 01-6.13-.75 15.72 15.72 0 01-5.1-3.57 15.4 15.4 0 01-3.17-5.27c-.79-1.9-2.11-3.78-2.11-6a16.43 16.43 0 011.55-6.2 16.46 16.46 0 013.3-5.41c1.48-1.49 3.68-2 5.65-2.79a15.21 15.21 0 016-1.41 15.2 15.2 0 016 1.39c2 .82 4 1.49 5.48 3a15.92 15.92 0 013.46 5.28 16.19 16.19 0 011.4 6.18 15.94 15.94 0 01-1.7 6.05z"
              />
              <path
                d="M29.11 34.4l-2.16 1-10.88-5.32a15.83 15.83 0 00-.21 2l9.83 4-2.17 1-7.66-3.73a16.8 16.8 0 00.19 1.89l6.21 2.51-2.16 1-3.66-1.78a14.55 14.55 0 002.22 4.5l12-5.94 11.89 6a14.67 14.67 0 002.28-4.58l-3.66 1.78-2.16-1 6.2-2.52a14.42 14.42 0 00.2-1.88L37.72 37l-2.16-1 9.82-4a12.77 12.77 0 00-.21-2L34.3 35.37l-2.16-1L45 29.19a13.08 13.08 0 00-.58-1.87l-13.66 5.25-14.22-4.39c-.11.33-.2.67-.28 1z"
              />
              <path
                d="M30.76 31.19l4.81-1.85a6 6 0 00-10 .24zM36.48 46.27a15.48 15.48 0 002.58-1.44l-8.46-3.62-8.48 3.57a14.75 14.75 0 002.6 1.47l5.92-2.94z"
              />
              <path
                d="M42 42.14l-11.4-4.87-11.41 4.79a14.4 14.4 0 002 2l9.47-4.7L40 44.11a14.74 14.74 0 002-1.97zM30.6 45.16l-4.11 1.72a14.89 14.89 0 004.13.6 14.7 14.7 0 004.06-.58zM30.76 19.72c4 0 8 2.47 11.5 7l1.58-.61a14.78 14.78 0 00-26.82.8l1.66.51c3.44-4.98 7.71-7.7 12.08-7.7z"
              />
              <path
                d="M30.76 20.82c-3.91 0-7.78 2.45-11 6.91l1.95.6c2.56-3.65 5.72-5.68 9-5.68 3 0 6 1.86 8.56 5.2l1.95-.75c-3.22-4.05-6.92-6.28-10.46-6.28z"
              />
              <path
                d="M30.68 23.47c-2.9 0-5.75 1.82-8.13 5.11l1.7.53a7.3 7.3 0 0112.55-.32l1.64-.63c-2.39-3.02-5.13-4.69-7.76-4.69z"
              />
            </g>
          </g>
        </svg>
      </div>
      <div
        class="c2"
      >
        <div>
          <span>
            Join us
          </span>
          <ul>
            <li>
              <a
                href="/"
              >
                Tailor sign up
              </a>
            </li>
            <li>
              <a
                href="/"
              >
                Fabric seller sign up
              </a>
            </li>
          </ul>
        </div>
        <div>
          <span>
            Legal
          </span>
          <ul>
            <li>
              <a
                href="/"
              >
                Terms and conditions
              </a>
            </li>
            <li>
              <a
                href="/"
              >
                Privacy
              </a>
            </li>
            <li>
              <a
                href="/"
              >
                Cookies
              </a>
            </li>
            <li>
              <a
                href="/"
              >
                Site map
              </a>
            </li>
          </ul>
        </div>
      </div>
      <div
        class="c3"
      >
        <div>
          <span>
            Follow us
          </span>
          <ul>
            <li>
              <a
                href="https://www.facebook.com/fiveyardsapp"
              >
                <svg
                  class="svg-inline--fa fab-facebook-f fa-2x"
                  color="var(--color-white)"
                />
              </a>
            </li>
            <li>
              <a
                href="https://twitter.com/fiveyardsapp"
              >
                <svg
                  class="svg-inline--fa fab-twitter fa-2x"
                  color="var(--color-white)"
                />
              </a>
            </li>
            <li>
              <a
                href="https://www.instagram.com/fiveyardsapp"
              >
                <svg
                  class="svg-inline--fa fab-instagram fa-2x"
                  color="var(--color-white)"
                />
              </a>
            </li>
          </ul>
        </div>
        <div>
          <svg
            viewBox="0 0 142 53"
            xmlns="http://www.w3.org/2000/svg"
          >
            <defs>
              <clippath>
                <path
                  d="M0 0h105v39H0z"
                  fill="none"
                />
              </clippath>
              <clippath
                id="b"
              >
                <path
                  clip-rule="evenodd"
                  d="M14.873-53h92a5.873 5.873 0 015.873 5.873v27.405a5.873 5.873 0 01-5.873 5.873h-92A5.873 5.873 0 019-19.722v-27.405A5.873 5.873 0 0114.873-53z"
                  fill="none"
                  transform="translate(-9 53)"
                />
              </clippath>
              <clippath
                id="c"
              >
                <path
                  d="M1-2.275h119.408V-61H1z"
                  fill="none"
                  transform="translate(-1 61)"
                />
              </clippath>
              <clippath
                id="d"
              >
                <path
                  d="M0 0h123v60H0z"
                  fill="none"
                  transform="translate(-.067 -.067)"
                />
              </clippath>
              <clippath
                id="e"
              >
                <path
                  d="M9-13.85h103.748V-53H9z"
                  fill="none"
                  transform="translate(-9 53)"
                />
              </clippath>
              <clippath
                id="f"
              >
                <path
                  clip-rule="evenodd"
                  d="M14.873-53H48.15v39.15H14.873A5.873 5.873 0 019-19.722v-27.405A5.873 5.873 0 0114.873-53z"
                  fill="none"
                  transform="translate(-9 53)"
                />
              </clippath>
              <clippath
                id="g"
              >
                <path
                  d="M28.959-47l.444 1.437h1.437l-1.162.888.444 1.437-1.162-.889-1.162.889.444-1.437-1.163-.888h1.437z"
                  fill="none"
                  transform="translate(-27.078 47)"
                />
              </clippath>
              <clippath
                id="h"
              >
                <path
                  d="M28.959-22.843l.444 1.437h1.437l-1.162.888.444 1.436-1.162-.888-1.162.888.444-1.436-1.163-.888h1.437z"
                  fill="none"
                  transform="translate(-27.078 22.843)"
                />
              </clippath>
              <clippath
                id="i"
              >
                <path
                  d="M35-44.8l.444 1.437h1.437l-1.163.888.444 1.436L35-41.931l-1.162.888.444-1.436-1.162-.888h1.437z"
                  fill="none"
                  transform="translate(-33.118 44.804)"
                />
              </clippath>
              <clippath
                id="j"
              >
                <path
                  d="M39.881-29l.444 1.437h1.437l-1.162.888.444 1.436-1.162-.888-1.162.888.444-1.436L38-27.563h1.437z"
                  fill="none"
                  transform="translate(-38 29)"
                />
              </clippath>
              <clippath
                id="k"
              >
                <path
                  d="M41.038-34.922l.444 1.437h1.437l-1.162.888.443 1.437-1.162-.888-1.162.888.444-1.44-1.163-.888h1.437z"
                  fill="none"
                  transform="translate(-39.157 34.922)"
                />
              </clippath>
              <clippath
                id="l"
              >
                <path
                  d="M16.881-34.922l.444 1.437h1.437L17.6-32.6l.444 1.437-1.162-.888-1.163.888.444-1.437L15-33.485h1.437z"
                  fill="none"
                  transform="translate(-15 34.922)"
                />
              </clippath>
              <clippath
                id="m"
              >
                <path
                  d="M17.881-29l.444 1.437h1.437l-1.162.888.444 1.436-1.162-.888-1.162.888.444-1.436L16-27.563h1.437z"
                  fill="none"
                  transform="translate(-16 29)"
                />
              </clippath>
              <clippath
                id="n"
              >
                <path
                  d="M39.881-41l.444 1.437h1.437l-1.162.888.444 1.436-1.162-.888-1.162.888.444-1.436L38-39.563h1.437z"
                  fill="none"
                  transform="translate(-38 41)"
                />
              </clippath>
              <clippath
                id="o"
              >
                <path
                  d="M18.381-41l.444 1.437h1.437l-1.162.888.444 1.436-1.162-.888-1.162.888.444-1.436-1.164-.888h1.437z"
                  fill="none"
                  transform="translate(-16.5 41)"
                />
              </clippath>
              <clippath
                id="p"
              >
                <path
                  d="M36.1-23.941l.44 1.441h1.437l-1.163.888.444 1.437-1.158-.892-1.163.888.444-1.437-1.165-.884h1.437z"
                  fill="none"
                  transform="translate(-34.216 23.941)"
                />
              </clippath>
              <clippath
                id="q"
              >
                <path
                  d="M22.92-44.8l.444 1.437H24.8l-1.162.888.444 1.436-1.163-.888-1.162.888.444-1.436-1.162-.888h1.437z"
                  fill="none"
                  transform="translate(-21.039 44.804)"
                />
              </clippath>
              <clippath
                id="r"
              >
                <path
                  d="M21.822-23.941l.444 1.437H23.7l-1.162.888.444 1.437-1.162-.887-1.162.887.444-1.437-1.161-.884h1.437z"
                  fill="none"
                  transform="translate(-19.941 23.941)"
                />
              </clippath>
              <clippath
                id="s"
              >
                <path
                  clip-rule="evenodd"
                  d="M56-37.5a4.444 4.444 0 004.718 4.5 5.041 5.041 0 003.856-1.7v-3.4h-4.209v1.864h2v.73a2.719 2.719 0 01-1.647.522 2.4 2.4 0 01-2.431-2.516 2.4 2.4 0 012.431-2.516 2.167 2.167 0 011.843 1.03l1.882-.991A4.1 4.1 0 0060.718-42 4.444 4.444 0 0056-37.5zm9.894 4.342v-8.7h3.659c2.745 0 4.718 1.643 4.718 4.342S72.3-33.16 69.566-33.16zm2.248-1.956h1.412a2.312 2.312 0 002.418-2.4 2.23 2.23 0 00-2.4-2.386h-1.431zm7.254 1.956v-8.7h4.364a2.81 2.81 0 013.032 2.894 2.788 2.788 0 01-3.032 2.855h-2.117v2.947zm5.11-5.815a.974.974 0 00-1.046-.991h-1.817v1.956h1.817a.961.961 0 001.045-.967zm8.154 5.813l-1.46-2.947h-1.15v2.947H83.8v-8.7h4.365a2.8 2.8 0 013.035 2.898 2.635 2.635 0 01-1.738 2.608l1.777 3.194zm.248-5.828a1 1 0 00-1.085-.978h-1.776v1.956h1.777a1 1 0 001.085-.978z"
                  fill="none"
                  transform="translate(-56 42)"
                />
              </clippath>
              <clippath
                id="t"
              >
                <path
                  d="M31.991-33.952h-.419v-1.3A2.706 2.706 0 0028.915-38a2.706 2.706 0 00-2.657 2.747v1.3h-.419a.854.854 0 00-.839.868v3.47a.854.854 0 00.839.868h6.152a.854.854 0 00.839-.868v-3.47a.854.854 0 00-.839-.867zm-1.818 0h-2.516v-1.3a1.282 1.282 0 011.258-1.3 1.282 1.282 0 011.258 1.3z"
                  fill="none"
                  transform="translate(-25 38)"
                />
              </clippath>
            </defs>
            <g>
              <g>
                <g
                  clip-path="url(#a)"
                >
                  <g
                    clip-path="url(#b)"
                  >
                    <g
                      clip-path="url(#c)"
                      transform="translate(-7.83 -7.83)"
                    >
                      <g>
                        <g
                          clip-path="url(#d)"
                          transform="translate(-2.103 -2.103)"
                        >
                          <g
                            clip-path="url(#b)"
                            transform="translate(9.933 9.933)"
                          >
                            <g
                              clip-path="url(#e)"
                            >
                              <path
                                d="M5.873 0h92a5.873 5.873 0 015.873 5.873v27.405a5.873 5.873 0 01-5.873 5.873h-92A5.873 5.873 0 010 33.278V5.873A5.873 5.873 0 015.873 0z"
                                fill="#fff"
                              />
                            </g>
                          </g>
                          <g
                            clip-path="url(#e)"
                            transform="translate(9.933 9.933)"
                          >
                            <g>
                              <path
                                d="M5.873 0h92a5.873 5.873 0 015.873 5.873v27.405a5.873 5.873 0 01-5.873 5.873h-92A5.873 5.873 0 010 33.278V5.873A5.873 5.873 0 015.873 0z"
                                fill="none"
                                stroke="#f3f3f3"
                                stroke-width="2"
                              />
                            </g>
                          </g>
                        </g>
                      </g>
                    </g>
                  </g>
                  <g
                    clip-path="url(#f)"
                  >
                    <g
                      clip-path="url(#c)"
                      transform="translate(-7.83 -7.83)"
                    >
                      <path
                        d="M2.936 2.936h48.938v48.938H2.936z"
                        fill="#1f4fdf"
                      />
                    </g>
                  </g>
                  <g
                    clip-path="url(#g)"
                    transform="translate(17.694 5.873)"
                  >
                    <g
                      clip-path="url(#c)"
                      transform="translate(-25.524 -13.703)"
                    >
                      <path
                        d="M20.63 8.809h13.549v13.549H20.63z"
                        fill="#f7d02e"
                      />
                    </g>
                  </g>
                  <g
                    clip-path="url(#h)"
                    transform="translate(17.694 29.516)"
                  >
                    <g
                      clip-path="url(#c)"
                      transform="translate(-25.524 -37.346)"
                    >
                      <path
                        d="M20.63 32.453h13.549v13.549H20.63z"
                        fill="#f7d02e"
                      />
                    </g>
                  </g>
                  <g
                    clip-path="url(#i)"
                    transform="translate(23.605 8.022)"
                  >
                    <g
                      clip-path="url(#c)"
                      transform="translate(-31.435 -15.852)"
                    >
                      <path
                        d="M26.542 10.962h13.549v13.549H26.542z"
                        fill="#f7d02e"
                      />
                    </g>
                  </g>
                  <g
                    clip-path="url(#j)"
                    transform="translate(28.384 23.49)"
                  >
                    <g
                      clip-path="url(#c)"
                      transform="translate(-36.214 -31.32)"
                    >
                      <path
                        d="M31.32 26.426h13.549v13.549H31.32z"
                        fill="#f7d02e"
                      />
                    </g>
                  </g>
                  <g
                    clip-path="url(#k)"
                    transform="translate(29.516 17.694)"
                  >
                    <g
                      clip-path="url(#c)"
                      transform="translate(-37.346 -25.524)"
                    >
                      <path
                        d="M32.453 20.63h13.549v13.549H32.453z"
                        fill="#f7d02e"
                      />
                    </g>
                  </g>
                  <g
                    clip-path="url(#l)"
                    transform="translate(5.873 17.694)"
                  >
                    <g
                      clip-path="url(#c)"
                      transform="translate(-13.703 -25.524)"
                    >
                      <path
                        d="M8.809 20.63h13.549v13.549H8.809z"
                        fill="#f7d02e"
                      />
                    </g>
                  </g>
                  <g
                    clip-path="url(#m)"
                    transform="translate(6.851 23.49)"
                  >
                    <g
                      clip-path="url(#c)"
                      transform="translate(-14.681 -31.32)"
                    >
                      <path
                        d="M9.788 26.426h13.549v13.549H9.788z"
                        fill="#f7d02e"
                      />
                    </g>
                  </g>
                  <g
                    clip-path="url(#n)"
                    transform="translate(28.384 11.745)"
                  >
                    <g
                      clip-path="url(#c)"
                      transform="translate(-36.214 -19.575)"
                    >
                      <path
                        d="M31.32 14.681h13.549V28.23H31.32z"
                        fill="#f7d02e"
                      />
                    </g>
                  </g>
                  <g
                    clip-path="url(#o)"
                    transform="translate(7.341 11.745)"
                  >
                    <g
                      clip-path="url(#c)"
                      transform="translate(-15.171 -19.575)"
                    >
                      <path
                        d="M10.277 14.681h13.549V28.23H10.277z"
                        fill="#f7d02e"
                      />
                    </g>
                  </g>
                  <g
                    clip-path="url(#p)"
                    transform="translate(24.68 28.441)"
                  >
                    <g
                      clip-path="url(#c)"
                      transform="translate(-32.51 -36.271)"
                    >
                      <path
                        d="M27.616 31.378h13.549v13.549H27.616z"
                        fill="#f7d02e"
                      />
                    </g>
                  </g>
                  <g
                    clip-path="url(#q)"
                    transform="translate(11.783 8.022)"
                  >
                    <g
                      clip-path="url(#c)"
                      transform="translate(-19.613 -15.852)"
                    >
                      <path
                        d="M14.719 10.962h13.549v13.549H14.719z"
                        fill="#f7d02e"
                      />
                    </g>
                  </g>
                  <g
                    clip-path="url(#r)"
                    transform="translate(10.709 28.441)"
                  >
                    <g
                      clip-path="url(#c)"
                      transform="translate(-18.539 -36.271)"
                    >
                      <path
                        d="M13.645 31.378h13.549v13.549H13.645z"
                        fill="#f7d02e"
                      />
                    </g>
                  </g>
                  <g
                    clip-path="url(#s)"
                    transform="translate(46.001 10.766)"
                  >
                    <g
                      clip-path="url(#c)"
                      transform="translate(-53.831 -18.596)"
                    >
                      <path
                        d="M48.938 13.703h45.023v18.784H48.938z"
                        fill="#2b3844"
                      />
                    </g>
                  </g>
                  <g
                    clip-path="url(#t)"
                    transform="translate(15.66 14.681)"
                  >
                    <g
                      clip-path="url(#c)"
                      transform="translate(-23.49 -22.511)"
                    >
                      <path
                        d="M18.596 17.618h17.618v19.041H18.596z"
                        fill="#fff"
                      />
                    </g>
                  </g>
                  <g
                    clip-path="url(#c)"
                    transform="translate(-7.83 -7.83)"
                  >
                    <text
                      fill="#2b3844"
                      font-family="HelveticaNeue-Medium, Helvetica Neue"
                      font-size="6"
                      font-weight="500"
                      letter-spacing=".025em"
                      transform="translate(53.83 35.83)"
                    >
                      <tspan
                        x="0"
                        y="0"
                      >
                        COMPLIANT
                      </tspan>
                    </text>
                  </g>
                </g>
              </g>
            </g>
          </svg>
        </div>
      </div>
    </div>
    <div
      class="c4"
    >
      © 
      2020
       Five Yards
    </div>
  </footer>
</div>
`;
